### ISSUE TITLE
Add source files with multiple contests (contestIds) to test that this is handled properly

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#477]

**Comment by HEdingfield** [comment]
@chughes297 or @gngilbert, would you mind putting together something like this for us? Clear Ballot would probably be easiest, but if we could get them for Dominion and Hart too, that'd be a bonus.

**Comment by tarheel** [comment]
Ideally we would construct a single test that has one source from each of the non-CDF providers we support (Clear Ballot, Dominion, Hart, ES&S)!

**Comment by gngilbert** [comment]
By "source files" I presume you mean test data sets.  Can you point me to the current files you have tested for Clear Ballot, Dominion and Hart so that I will know that I am working off of the same versions you are?

**Comment by HEdingfield** [comment]
They're [here](https://github.com/BrightSpots/rcv/tree/develop/src/test/resources/network/brightspots/rcv/test_data) under the clear_ballot_*, dominion_*, and hart_* folders, respectively.

**Comment by gngilbert** [comment]
Thanks.

**Comment by gngilbert** [comment]
Sorry but I am unfamiliar with GitHub and can't figure out how to download
a file to my laptop.  Assuming that is what I need to do.  (Starting with
Clear Ballot)

George Gilbert
RCV Resource Center
george.gilbert@rankedchoicevoting.org
336-906-0047

**Comment by gngilbert** [comment]
Ok.  I got the cvr.csv for the kansas primary for Clear Ballot.  Is this the only file I need to work with to create a test data set for Clear Ballot?  

Looking back at the first set of Clear ballot test files we got (10/31/19), I see four additional files: choices.csv, contest.csv, precincts.csv and parties.csv.  ??

**Comment by tarheel** [comment]
@moldover can best answer these questions.

**Comment by HEdingfield** [comment]
It'd be nice to have a CDF JSON (and XML once that's in), with multiple contests too.

**Comment by moldover** [comment]
@gngilbert: Yes, you only need the cvr file.   The additional files are useful for verification.  We can also add a feature to slurp up the candidates (choices) data from the choices file automatically so the user doesn't have to enter them manually into the config.

You can use the Kansas csv as a starting point to create a multi-contest cvr: by extending the columns with a new contest.  The column headers would look like My_Choice_ID:My_Contest_ID:Rank:Vote For 1:Candidate_ID:Party_ID.

**Comment by gngilbert** [comment]
Okay. Thanks. I will give it a try.

**Comment by chughes297** [comment]
have we made any progress on this? or do we need to pick this back up? we do need this feature (not realistic for 1.3.0 but would be good to have for next version)

**Comment by tarheel** [comment]
We support this, right? I think this issue is just flagging that we need to have an automated test to confirm that it works. @moldover should have better context here.
